packages/front-end/components/GeneralSettings/SavedGroupSettings.tsx

```diff
@@ -0,0 +1,43 @@
+import { Box, Flex, Heading, Text } from "@radix-ui/themes";
+import Field from "@/components/Forms/Field";
+import { ConnectSettingsForm } from "@/pages/settings";
+import Frame from "@/ui/Frame";
+import { hasFileConfig } from "@/services/env";
+
+export default function SavedGroupSettings() {
+  return (
+    <ConnectSettingsForm>
+      {({ register }) => (
+        <Frame>
+          <Flex gap="4">
+            <Box width="220px" flexShrink="0">
+              <Heading size="4" as="h4">
+                Saved Group Settings
+              </Heading>
+            </Box>
+
+            <Flex align="start" direction="column" flexGrow="1" pt="6">
+              <Box mb="6" width="100%" mt="2">
+                <Text as="label" className="font-weight-semibold" size="3">
+                  ID List Size Limit
```

Comment:
Small nit: Wonder if it would make sense to add a tooltip here to outline why someone might want to set this value? I'll defer to you as you're closer to the issue this is solving than I am.